Main/Source/gods.cpp

```diff
@@ -126,15 +129,166 @@ int mortifer::GetBasicAlignment() const { return EVIL; }
+  FAVOUR_FIXEQUIPMENT,
+  FAVOUR_HEALBURNS,
+  FAVOUR_HOLYGREN,
+  FAVOUR_INFRAVISION,
+  FAVOUR_INVIGORATE,
+  FAVOUR_INVISIBILITY,
+  FAVOUR_SHOPPING,
+  FAVOUR_SPEEDUP,
+  FAVOUR_STOPFIRE,
+  FAVOUR_SUMMONWOLF,
+  FAVOUR_TAME,
+  FAVOUR_TELEPORT,
+  FAVOUR_BURNENEMIES,
+  FAVOUR_FEELENEMIES,
+  FAVOUR_POLYCONTROL,
+  FAVOUR_TELEPCONTROL,
+  FAVOUR_ENRAGE,
+  FAVOUR_CAUSEFEAR,
+};
+
+void god::FavourInit() //this one is better on this file
+{
+  AddFavourID(FAVOUR_BURNENEMIES,"Burn your Enemies");
+  AddFavourID(FAVOUR_CALLRAIN,"Make it Rain");
+  AddFavourID(FAVOUR_CAUSEFEAR,"Your enemies will Fear you");
+  AddFavourID(FAVOUR_CONFUSE,"Cause Confusion amongst your enemies");
+  AddFavourID(FAVOUR_CURELEPROSY,"Cure Leprosy");
+  AddFavourID(FAVOUR_CURELYCANTHROPY,"Cure Lycanthropy");
+  AddFavourID(FAVOUR_CUREMINDWORM,"Cure Mindworm");
+  AddFavourID(FAVOUR_CUREPOISON,"Cure Poison");
+  AddFavourID(FAVOUR_CURESLOWNESS,"Cure Slowness");
+  AddFavourID(FAVOUR_CURETAPEWORM,"Cure Tapeworm");
+  AddFavourID(FAVOUR_CUREWOUNDS,"Cure Wounds");
+  AddFavourID(FAVOUR_DISEASEIMMUNITY,"Gain temporary Immunity to Diseases");
+  AddFavourID(FAVOUR_EARTHQUAKE,"Invoke the rage of an Earth Quake");
+  AddFavourID(FAVOUR_ENCHANT,"Enchant Equipment");
+  AddFavourID(FAVOUR_ENRAGE,"Fill you with Rage");
+  AddFavourID(FAVOUR_ETHEREALMOV,"Become Ethereal");
+  AddFavourID(FAVOUR_EXTINGUISHFIRE,"Put out these Flames"); //TODO consider price vs FAVOUR_HEALBURNS);
+  AddFavourID(FAVOUR_FEED,"Calms your Hunger");
+  AddFavourID(FAVOUR_FEELENEMIES,"Feel your Enemies");
+  AddFavourID(FAVOUR_FIRESTORM,"Fiery Firestorm");
+  AddFavourID(FAVOUR_FIXEQUIPMENT,"Fix one broken equipped item");
+  AddFavourID(FAVOUR_HEALBURNS,"Heals your burns");
+  AddFavourID(FAVOUR_HOLYGREN,"Paladin's Holy Grenade");
+  AddFavourID(FAVOUR_INFRAVISION,"See in the Darkness");
+  AddFavourID(FAVOUR_INVIGORATE,"Invigorate");
+  AddFavourID(FAVOUR_INVISIBILITY,"Become Invisible");
+  AddFavourID(FAVOUR_POLYCONTROL,"Control what you are");
+  AddFavourID(FAVOUR_SHOPPING,"Black Friday");
+  AddFavourID(FAVOUR_SPEEDUP,"Make you Fast");
+  AddFavourID(FAVOUR_STOPFIRE,"Unburn one Equipment");
+  AddFavourID(FAVOUR_SUMMONWOLF,"Summon Wolf friend(s)");
+  AddFavourID(FAVOUR_TAME,"Tame this Monster");
+  AddFavourID(FAVOUR_TELEPCONTROL,"Decide where you are sent");
+  AddFavourID(FAVOUR_TELEPORT,"Teleport");
+}
+
+int CalcDebit(god* G,int iDebit,int iDefault){
+  if(iDebit!=0){
+    switch(iDebit){
+      case FAVOURDEBIT_AUTO:       iDebit=iDefault  ;break;
+      case FAVOURDEBIT_AUTOHALF:   iDebit=iDefault/2;break;
+      case FAVOURDEBIT_AUTODOUBLE: iDebit=iDefault*2;break;
+    }
+    
+    if(G->GetRelation() < 0) //costy if bad relation
+      iDebit*=2;
+    
+    // can ask more favours if very well aligned
+    if(game::GetPlayerAlignment() == game::GetGodAlignmentVsPlayer(G)){
+      iDebit/=2;
+    }
+    
+    // skilled in praying :)
+    iDebit -= game::GetPlayer()->GetAttribute(MANA);
```

Comment:
Maybe WISDOM? I know it's already used for normal prayers, but it could still use a bit of a boost in usefulness, plus it's already associated with communing with gods.

Reply:
mmm... what weight? may be
`(MANA*7+WISDOM*3)/10` or `MANA + WISDOM/2`?
MANA is the MagickPower for what i understood everywhere it is used, I am not sure how to use wisdom here.
Initially I thought on MANA changing the percentual of the debit, but later I thought it better not lower too much the debit, so there is also the minimum (for a master prayer :> )

Reply:
I havent changed this about wisdom yet.

also I couldnt test all of these other changes yet...
they just compiled so I uploaded to let it be tested :>

Reply:
I just thout MANA should be called something like 
MagicAttunement or MagicalAcclimatization
Mana always make me think of "Mana pool" since I begin playing this was very confusing like "where is my mana pool?" "How I spend it? using wands?" but til now i sounds weird having Mana there :P

Reply:
MANA was so far used for manipulating magic items, while WISDOM is used to persuade gods to give you a better prayer outcome. I think that's quite appropriate for similarly persuading the gods to give you lower debit when asking for a favour.

As you mentioned, some weighted combination of MANA and WISDOM could be used, too. :)

> Initially I thought on MANA changing the percentual of the debit, but later I thought it better not lower too much (or too fast) the price, so there is also the minimum (in case player becomes a master prayer :> )

Yeah, wouldn't want the favours to feel cheap. :+1:

Reply:
mmm... so a manipulative praying (AKA Favour) would require Mana and Wisdom xD
I don't know if Mana can be easily improved, on the other hand, wisdom goes up "easily" thru ommel products and reading books right? so for now I will put it like: `((MANA*2)+WISDOM)/3`